deploy/service.yaml

```diff
@@ -130,11 +130,11 @@ spec:
       labels:
         run: prometheus-service
         app.kubernetes.io/name: prometheus-service
-        app.kubernetes.io/version: 0.6.1
+        app.kubernetes.io/version: latest
     spec:
       containers:
         - name: prometheus-service
-          image: keptncontrib/prometheus-service:0.6.1
+          image: keptncontrib/prometheus-service:latest
```

Comment:
shouldn't we use a fixed version here?

Reply:
Good catch, I set it to `prometheus-service:0.6.3-dev-PR-177.202110010812` which is the latest image version available.